oxidation/libparsec/crates/crypto/src/sodiumoxide/sign.rs

```diff
@@ -113,14 +113,10 @@ impl VerifyKey {
     /// Verify a signature using the given [VerifyKey], `signature` and `message`
     pub fn verify_with_signature(
         &self,
-        raw_signature: &[u8],
+        raw_signature: [u8; Signature::BYTE_SIZE],
```

Comment:
use instead `SigningKey::SIGNATURE_SIZE` (this should be the same value, but it's more clear to use the same constant everywhere, and if it endup not being the same value... we've detected a bug !)